tools/validators/instance_validator/validate/instance_parser.py

```diff
@@ -33,7 +33,19 @@
 """Schema separately parses translation to account for multiple valid formats
 github.com/google/digitalbuildings/blob/master/ontology/docs/building_config.md
 #defining-translations"""
-_TRANSLATION_SCHEMA = syaml.Str() | syaml.Any()
+_TRANSLATION_SCHEMA = syaml.Str() | syaml.MapPattern(
+    syaml.Str(),
```

Comment:
doesn't the external pipe invalidate the stricter validation?

Reply:
the Str is for the key word `Compliant` which is the case with `translation: compliant`
Strictyaml does not allow you to do this:

`Compliant` | | syaml.MapPattern( ...

See output error in the description below on why we need to validate Comlpiant seperatly